exercises/resistor-color-duo/src/main/kotlin/ResistorColorDuo.kt

```diff
@@ -0,0 +1,6 @@
+object ResistorColorDuo {
+
+    fun colorCode(vararg input: String): Int {
```

Comment:
Let's discuss the signature.  
The problem description states that here we must implement a function that _decodes color code_ to a numeric _resistance value_. Such a query should most probably be named after the characteristic it queries, say 
```kotlin
fun resistance(vararg colors: String): Int
```
for domain-specific query, or
```kotlin
fun value(vararg colors: String): Int
```
for general-purpose one

What do you think?